backend/target_zone_estimation.py

```diff
@@ -43,6 +33,17 @@ async def handle_data_streaming(websocket):
             await asyncio.sleep(1)
         except Exception as e:
             print(f"Error occurred during data handling: {e}")
+async def load_data_from_file(file_path):
+    """Load force data from a TSV file."""
+    with open(file_path, "r") as file:
+        # Skip headers if present
+        for line in file:
+            if line.startswith("SAMPLE") or line.strip() == "":
```

Comment:
I took a look at a sample .tsv and the files start with NO_OF_SAMPLES, FREQUENCY, TIMESTAMP etc. and other lines similar to this before the line outlining the sample data format (specifically the line that starts like: SAMPLE 
   TIME   FORCE_X,...) you handle with the if statement. I'm not sure you handle this case here.